tests/features/community_content/notification.post_moderated.discussion.feature

```diff
@@ -86,7 +86,7 @@ Feature: Notification test for the discussion transitions on a post moderated pa
     Then the following email should have been sent:
       | recipient | CC member                                                                                                                                                        |
       | subject   | Joinup: Content has been updated                                                                                                                                 |
-      | body      | the Facilitator, CC Facilitator has approved your request to publish the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
+      | body      | the Facilitator, CC Facilitator has approved your request of publication of the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
```

Comment:
Hmm I think this was better before, I guess this is the typical case where the content editors speak 5 languages but none of them correctly. I am starting to think that having awkward grammar is a hard requirement for the commission. If someone ever receives a mail from the EC which is correctly formatted and spelled then they will distrust it and think it is a phishing attempt.